### PR TITLE
List-group styling first draft - Based on 3.5.x

### PR DESCRIPTION
**Pull request by mmexvr**
**Describe pull-request**</br>
_Intial styling for List-group_

- border-radius
- padding
- Active color
- Border-color

**Solving issue**</br>
Fixes a part of list-group in: #11 and #32 

**Additional context**<br/>
- Based on 3.5.x
- No styling done for the action items

### CODE REVIEW

**Comment by mmexvr** [comment]
Conflict solved

After this is merged, update storybook and add list-groups

File: src/styles/components/_list-group.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@import '../variables';
+
+$border-radius-list-group: 3px;
```
Comment (helloimela) [inline src/styles/components/_list-group.scss]: It's the same with global border-radius
We can move this variable to _variable.scss

Comment (mmexvr) [reply src/styles/components/_list-group.scss]: Fixed

**Review by helloimela — APPROVED** [review]
Looks good!
Fix conflict and we can merge this one

**Pull request merged**